lib/stat_tracker.rb

```diff
@@ -6,18 +6,38 @@
 require './lib/team_info'
 
 class StatTracker
-  attr_reader :games
+  attr_reader :games, :teams, :game_team_stats
 
-  @games = []
-  @teams = []
-  @team_info_path = './data/team_info.csv'
+  def initialize(locations)
```

Comment:
Adding the get_* methods as part of the initialize was the missing key to making this work.